evergreen/run-tests.sh

```diff
@@ -25,9 +27,17 @@ provision_ssl () {
   uri_environment_variable_name=$1
   # Arguments for auth + SSL
   if [ "$AUTH" != "noauth" ] || [ "$TOPOLOGY" == "replica_set" ]; then
-    export $uri_environment_variable_name="${!uri_environment_variable_name}&ssl=true"
+    if [ "$OCSP_TLS_SHOULD_SUCCEED" != "nil" ]; then
+      export $uri_environment_variable_name="${!uri_environment_variable_name}&ssl=true"
+    else
+      export $uri_environment_variable_name="${!uri_environment_variable_name}&ssl=true&tlsInsecure=true"
```

Comment:
Setting `tlsInsecure=true` because we have too many tests that spin up custom mongoclients and the x509gen certs in drivers-evergreen-toosl don't pass certificate revocation checking

Reply:
Using the new URI option instead.